Sources/Model/Localized.yml

```diff
@@ -2,390 +2,489 @@ default: en
     en: Export Set \"(title)\"
-    de: Exportiere das Set \"(title)\"
-
+    de: Exportiere das Set \"(title)\
+    it: Esporta set \"(title)\"
+    
 addSet:
     en: Add Set
     de: Set hinzufügen
+    it: Aggiungi Set
 
 sets:
     en: Sets
-    de: Sets
-
+    de: Setsde: 
+    it: Sets
+    
 newWindow:
     en: New Window
     de: Neues Fenster
+    it: Nuova finestra
 
 closeWindow:
     en: Close Window
     de: Fenster schliessen
+    it: Chiudi finestra
 
 about:
     en: About Memorize
     de: Info zu Memorize
+    it: Informazioni su Memorize
 
 mainMenu:
     en: Main Menu
     de: Hauptmenü
+    en: Menu principale
 
 viewMenu:
     en: View
     de: Ansicht
+    en: Vista
 
 filter:
     en: Filter...
     de: Filtern ...
+    it: Filtro...
 
 noFlashcards:
     en: No flashcards available
     de: Keine Karteikarten verfügbar
+    it: Nessuna flashcard disponibile
 
 generateExam:
     en: Generate a Preparation Exam
     de: Erstelle eine Vorbereitungsprüfung
+    it: Generare preparazione di un esame
 
 generateExamDescription(title):
     en: Spot last mistakes before your exam about \"(title)\"
     de: Finde letzte Fehler vor deiner Prüfung zu \"(title)\"
+    it: Individua gli ultimi errori prima dell'esame su \"(title)\"
 
 answerWithBack:
     en: Answer With Back
     de: Antworte mit der Rückseite
+    it: Rispondi con il retro
 
 numberOfQuestions:
     en: Number of Questions
     de: Anzahl Fragen
+    it: Numero di domande
 
 testDescription(count, total):
     en(total == "1"): Testing (count) of (total) Flashcard
     en: Testing (count) of (total) Flashcards
     de(total == "1"): Teste (count) von (total) Karteikarte
     de: Teste (count) von (total) Karteikarten
+    it:(total == "1"): Test (count) di (total) Flashcard
+    it: Test (count) di (total) Flashcard
 
 createTest:
     en: Create Test
     de: Prüfung erstellen
+    it: Crea Test
 
 solveTest:
     en: Solve Test
     de: Test lösen
+    it: Risolvi Test
 
 fullScore:
     en: Full Score!
     de: Volle Punktzahl!
+    it: Punteggio completo!
 
 score(score):
     en(score == "1"): (score) Point
     en: (score) Points
     de(score == "1"): (score) Punkt
     de: (score) Punkte
+    it:(score == "1"): (score) Punto
+    it: (score) Punti
 
 maxScore:
     en: Maximum Score
     de: Maximale Punktzahl
+    it: Punteggio massimo
 
 noScoreData:
     en: "-"
     de: "-"
+    it: "-"
 
 scoreInstructions:
     en: Scroll down to check your answers
     de: Überprüfe deine Antworten weiter unten
+    it: Scorri verso il basso per verificare le tue risposte
 
 scoreLabel:
     en: Score
     de: Punktzahl
+    it: Punteggio
 
 scoreRowLabel:
     en: Your Score
     de: Deine Punktzahl
+    it: Il tuo punteggio
 
 check:
     en: Check
     de: Überprüfen
+    it: Controlla
 
 finishTest:
     en: Finish Test
     de: Test beenden
+    it: Fine del test
 
 star:
     en: Star
     de: Stern
+    it: Stella
 
 tags:
     en: Tags
     de: Tags
-
```

Comment:
Here, the Italian translation is missing, instead, you added a 2nd English one. Replace "en: Tag" by "it: <Translation>" (note that it is "Tags" in the plural).